CredentialCardDetail.vue

```diff
@@ -0,0 +1,174 @@
+<template>
+  <q-card class="row br-credential-card">
+    <q-card-title class="text-center s-card-title">
+      Credential Details
+    </q-card-title>
+    <div class="row justify-between s-card-info">
+      <div class="column items-center s-card-info-left">
+        <q-card-media>
+          <q-icon
+            v-if="useDefaultImage || !credential.issuerLogo & !credential.image"
```

Comment:
I don't think this does the right thing, see: https://repl.it/repls/GuiltySpotlessScans

Reply:
Updated.